plugins/crypto/pkcs11/pkcs11.h

```diff
@@ -0,0 +1,1765 @@
+/* pkcs11.h
```

Comment:
I don't like the "code dump".
The file is from here: https://github.com/OpenSC/libp11/blob/master/src/pkcs11.h
It would be much better if that library was installed system-wide.
So we don't have to maintain this going forward.